src/main/java/com/faforever/client/coop/CoopController.java

```diff
@@ -172,9 +177,7 @@ private String coopMissionFromFolderNamer(List<CoopMissionBean> coopMaps, String
   }
 
   private String commaDelimitedPlayerList(CoopResultBean coopResult) {
-    return coopResult.getReplay().getTeams().values().stream()
-        .flatMap(List::stream)
-        .collect(Collectors.joining(i18n.get("textSeparator")));
+    return String.join(", ", coopResult.getReplay().getTeams().values().stream().flatMap(List::stream).toList());
```

Comment:
Why was this changed to not use the internationalized separator? Also the Collectors.joining is the preferred method since we are already using the stream api.

Reply:
> Why was this changed to not use the internationalized separator?

Usually this only applies to numbers, doesn't it?

> Also the Collectors.joining is the preferred method since we are already using the stream API.

Weird. The IDE suggested that it's better to use this instead. Ok, I will fix it

Reply:
the text separator applies to text numbers is something different.